packages/libcunit/build.sh

```diff
@@ -0,0 +1,14 @@
+TERMUX_PKG_HOMEPAGE=https://sourceforge.net/projects/cunit/
+TERMUX_PKG_DESCRIPTION="C Unit Testing Framework"
+TERMUX_PKG_LICENSE="GPL-2.0"
+TERMUX_PKG_MAINTAINER="Thibault Meyer <meyer.thibault@gmail.com>"
+TERMUX_PKG_VERSION=2.1.3
+TERMUX_PKG_REVISION=1
```

Comment:
Please omit TERMUX_PKG_REVISION completely, it is not needed for new packages, or when TERMUX_PKG_VERSION is changed

Reply:
Removed